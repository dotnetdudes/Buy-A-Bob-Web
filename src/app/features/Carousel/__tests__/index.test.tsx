import { Carousel } from "..";
import { render } from "app/utils/test-utils";
import * as React from "react";

jest.mock("react-i18next", () => ({
  useTranslation: () => {
    return {
      t: (str) => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe("<Carousel  />", () => {
  it("should match snapshot", () => {
    const loadingIndicator = render(<Carousel />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
