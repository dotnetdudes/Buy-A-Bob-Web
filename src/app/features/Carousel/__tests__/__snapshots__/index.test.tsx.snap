// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Carousel  /> should match snapshot 1`] = `
<div>
  hey
</div>
`;
